.github/workflows/ci.yml

```diff
@@ -0,0 +1,112 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - 'v*'
+  pull_request: {}
+  schedule:
+    - cron:  '0 3 * * *' # daily, at 3am
+
+jobs:
+  #lint:
+  #  name: Linting
+  #  runs-on: ubuntu-latest
+  #
+  #  steps:
+  #    - uses: actions/checkout@v1
+  #    - uses: actions/setup-node@v1
+  #      with:
+  #        node-version: 10.x
+  #
+  #    - name: get yarn cache dir
+  #      id: yarn-cache
+  #      run: echo "::set-output name=dir::$(yarn cache dir)"
+  #
+  #    - uses: actions/cache@v1
+  #      with:
+  #        path: ${{ steps.yarn-cache.outputs.dir }}
+  #        key: ${{ runner.os }}-yarn-lint-${{ hashFiles('**/yarn.lock') }}
+  #        restore-keys: |
+  #          ${{ runner.os }}-yarn-
+  #
+  #    - name: install dependencies
+  #      run: yarn install
+  #
+  #    - name: lint
+  #      run: yarn lint
+
+  tests:
+    name: Tests
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        test-suite:
+          - test:one ember-1.13
+          - test:one ember-lts-2.4
+          - test:one ember-lts-2.8
+          - test:one ember-3.0
+          - test:one ember-3.0
+          - test:one ember-lts-3.4
+          - test:one ember-lts-3.8
+          - test:one ember-lts-3.12
+          - test:one ember-release
+          - test:one ember-beta
+          - test:one ember-canary
+        include:
+          - test-suite: "test:one ember-canary"
+            allow-failure: true
+
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 10.x
+
+      - name: get yarn cache dir
+        id: yarn-cache
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - uses: actions/cache@v1
+        with:
+          path: ${{ steps.yarn-cache.outputs.dir }}
+          key: ${{ runner.os }}-yarn-test-${{ hashFiles('**/yarn.lock') }}
```

Comment:
the `key` should probably include the matrix value, no?

Reply:
I don't think that's necessary since we're installing the dependencies from `yarn.lock` for **all** of the matrix values, then only in the actual build step would ember-try install the alternative dependencies.

Reply:
sure, but the cache is saved **after** ember-try does its thing, so the saved cache artifact will be different for each job.

Reply:
…which makes me realize that for CI it'd be great if there was something like

```
ember try:install ember-lts-3.8 --skip-cleanup
```

that would install the dependencies as needed for a particular scenario only without running anything so we could cache whatever that generates and then in the actual build step run

```
ember try:one ember-lts-3.8 --skip-cleanup
```

but that would reuse what was retrieved from cache?

Reply:
in that case, why not just run `ember test` after the `try:install`? 😉

Reply:
> sure, but the cache is saved after ember-try does its thing, so the saved cache artifact will be different for each job.

Ah, you're right – wasn't aware of that. So also what I suggest above isn't needed then.

Reply:
fixed